### PR TITLE
chore: add screenshots after installing kubernetes & aws from the registry

### PR DESCRIPTION
**Pull request by yashmehrotra**
Fixes: https://github.com/flanksource/docs/issues/155

### CODE REVIEW

File: mission-control/docs/registry/aws.md [hunk 1]
```diff
@@ -10,6 +10,24 @@ helm repo update
 helm install {release-name} flanksource/mission-control-aws
```
Comment (moshloop) [inline mission-control/docs/registry/aws.md:10]:
```suggestion
helm install mission-control-aws  flanksource/mission-control-aws
```

File: mission-control/docs/registry/aws.md [hunk 2]
```diff
@@ -10,6 +10,24 @@ helm repo update
...
 ```
 
+After running `helm install` you should get a success message:
```
Comment (moshloop) [inline mission-control/docs/registry/aws.md:13]: I think we should have a pre-requisites step in the beginning that outlines the ways of connecting:

1) IAM Roles for Service Accounts
2) Access Key

and a the IAM policy required

Comment (moshloop) [reply mission-control/docs/registry/aws.md:13]:
```


{
	"Version": "2012-10-17",
	"Statement": [
		{
			"Sid": "mission-control-config-role",
			"Effect": "Allow",
			"Action": [
				"acm:Describe*",
				"acm:Get*",
				"acm:List*",
				"cloudtrail:LookupEvents",
				"ec2:Describe*",
				"ecr:Describe*",
				"eks:ListClusters",
				"eks:Describe*",
				"rds:Describe*",
				"elasticfilesystem:Describe*",
				"elasticloadbalancing:Describe*",
				"sts:GetCallerIdentity"
				"config:BatchGetAggregateResourceConfig",
				"config:BatchGetResourceConfig",
				"config:Describe*",
				"config:Get*",
				"config:List*",
				"config:SelectAggregateResourceConfig",
				"config:SelectResourceConfig",
				"guardduty:Describe*",
				"guardduty:Get*",
				"guardduty:List*",
				"iam:GetAccountName",
				"iam:GetAccountSummary",
				"iam:GetGroup",
				"iam:GetGroupPolicy",
				"iam:GetInstanceProfile",
				"iam:GetLoginProfile",
				"iam:GetPolicy",
				"iam:GetRole",
				"iam:GetRolePolicy",
				"iam:GetUser",
				"iam:List*",
				"lambda:List*",
				"trustedadvisor:Describe*",
				"trustedadvisor:DownloadRisk",
				"trustedadvisor:Get*",
				"trustedadvisor:List*",
			],
			"Resource": "*"
		}
	]
}
```

File: mission-control/docs/registry/aws.md [hunk 3]
```diff
@@ -10,6 +10,24 @@ helm repo update
...
+NOTES:
+AWS scraper added
+```
+
+When you go to the catalog now, you can now see all the AWS Resources
+
+![AWS Catalog](/img/aws-registry-catalog-scraper.png)
+
+
 ### Values
```
Comment (moshloop) [inline mission-control/docs/registry/aws.md:31]: nameOverride and fullnameOverride we shouldn't document, 

I don't think we should include the full connection details table here, rather add an example, and then link to schema and EnvVar reference

Comment (moshloop) [reply mission-control/docs/registry/aws.md:31]: For cost reporting also include details on what needs to be setup with a link to the guide,  and an example +  the IAM policy required

e.g.

```
	{
			"Effect": "Allow",
			"Action": [
				"athena:GetQueryExecution",
				"athena:GetQueryResults",
				"athena:StartQueryExecution"
			],
			"Resource": "arn:aws:athena:eu-west-1:765618022540:workgroup/primary"
		},
```

File: mission-control/docs/registry/kubernetes.md [hunk 4]
```diff
@@ -10,6 +10,31 @@ helm repo update
 helm install {release-name} flanksource/mission-control-kubernetes
```
Comment (moshloop) [inline mission-control/docs/registry/kubernetes.md:10]:
```suggestion
helm install mission-control-kubernetes flanksource/mission-control-kubernetes
```

File: mission-control/docs/registry/kubernetes.md [hunk 5]
```diff
@@ -10,6 +10,31 @@ helm repo update
...
+LAST DEPLOYED: Thu Feb 14 19:00:32 2024
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Kubernetes topology and scraper added
+```
+
+When you go to the dashboard now, you can now see a cluster topology:
```
Comment (moshloop) [inline mission-control/docs/registry/kubernetes.md:26]: Can you check the queries and/or use AWS as those metrics don't look right. Also see https://github.com/flanksource/docs/blob/main/CONTRIBUTING.md#screenshots

File: mission-control/docs/registry/aws.md (hunk 1 above)
Comment (yashmehrotra) [reply mission-control/docs/registry/aws.md:10]: 👍🏽

**Comment by yashmehrotra** [comment]
@moshloop Have addressed all the comments

**Pull request merged**